src/kani-compiler/src/codegen_cprover_gotoc/codegen/typ.rs

```diff
@@ -754,7 +757,7 @@ impl<'tcx> GotocCtx<'tcx> {
         if self.use_slice_fat_pointer(mir_type) {
             let pointer_name = match mir_type.kind() {
                 ty::Slice(..) => self.ty_mangled_name(mir_type),
-                ty::Str => "str".intern(),
+                ty::Str => "refstr".intern(),
```

Comment:
I wonder if there is a better name we could use here.

Reply:
Great question. Any suggestion?